chapter4.Rmd

```diff
@@ -299,12 +299,12 @@ factor(some_vector, ordered = TRUE, levels = c("Level_1", "Level_2", ...))
 *** =instructions
 - Use the example code above to define `speed_factor`, that contains the speed information as an ordered factor. You can start from `speed_vector`, which is already created for you.
 - Print `speed_factor` to the console.
-- Use [`summary()`](http://www.rdocumentation.org/packages/base/functions/summary) to generate a summary of `speed_factor`: automagically, R prints the factor levels in the right order.
+- Use [`summary()`](http://www.rdocumentation.org/packages/base/functions/summary) to generate a summary of `speed_factor`: automatically, R prints the factor levels in the right order.
```

Comment:
Resolved - I went back and forth on this. The two keys are right next to each other and I didn't want to fail my duties so I went for the safe answer :) I switched it back!